app/views/episodes/index.html.erb

```diff
@@ -1,14 +1,46 @@
-<p style="color: green"><%= notice %></p>
```

Comment:
This page should probably set a title?  So you can distinguish between searches in your history.  (Maybe also include the podcast title, if you're on that page).